broker/thinBroker.go

```diff
@@ -688,19 +688,28 @@ func (tb *ThinBroker) NotifyLdContext(w rest.ResponseWriter, r *rest.Request) {
 		notifyEleDatamap := notifyElemtData.([]interface{})
 		notifyCtxResp := NotifyContextResponse{}
 		w.WriteJson(&notifyCtxResp)
+		Link := DEFAULT_CONTEXT
 		for _, data := range notifyEleDatamap {
 			notifyData := data.(map[string]interface{})
 			notifyData["@context"] = context
 			expand, _ := tb.ExpandData(notifyData)
 			sz := Serializer{}
 			deSerializedEntity, err := sz.DeSerializeEntity(expand)
 			if err != nil {
-				rest.Error(w, err.Error(), 400)
```

Comment:
it should return 400 instead of neglecting it. it might miss a important entity notification.